manifests/client/host_config_entry.pp

```diff
@@ -89,7 +89,8 @@
 #   the global host key database.
 #
 # @param gssapiauthentication  Specifies whether user authentication
-#   based on GSSAPI is allowed.
+#   based on GSSAPI is allowed. If the system is connected to an IPA domain,
+#   this will be set to true, based on the existance of the `ipa` fact.
```

Comment:
When the `ipa` fact is present, the module will behave _as if_ this parameter had been set to `true`―regardless of the parameter's value (this behavior cannot be overridden).